recipes/holopy/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+    export CFLAGS="-arch x86_64"
+    export FFLAGS="-static -ff2c -arch x86_64"
+    export LDFLAGS="-Wall -undefined dynamic_lookup -bundle -arch x86_64"
+    export LDFLAGS="-Wl,-search_paths_first -L$(pwd) $LDFLAGS"
```

Comment:
Please make sure the existing environment variables are preserved (e.g. `export CFLAGS="${CFLAGS} -arch x86_64"`).